### ISSUE TITLE
Yosemite 10.10.2

### ISSUE THREAD

**Issue by flp4** [RanbirAulakh/Asus-Q501LA-Hackintosh#7]
Looks like the latest release has broken the Intel graphics driver, no QE/QI and very slow screen updates. I read that the support for Intel graphics has changed on http://www.tonymacx86.com/yosemite-laptop-support/145427-fix-intel-hd4200-hd4400-hd4600-mobile-yosemite.html. I tried what they suggested or my interpretation of it but it did not help. Any suggestions?

Thanks,
Frank

**Comment by RanbirAulakh** [comment]
@flp4 sorry for the long delay/response. Unfortunately, I do not have any access to my laptop. Your best bet is try to create a new topic in Tonymacx86. They should be able to help you out.

Sorry.

**Comment by gvkt** [comment]
I have forked the original repository to fix it for Yosemite 10.10.2.

https://github.com/gvkt/Asus-Q501LA-Hackintosh

See Wiki for explanation of changes and instructions
https://github.com/gvkt/Asus-Q501LA-Hackintosh/wiki

**Comment by flp4** [comment]
I am glad you have something running. I used the new EFI.zip with 10.10.1 and the graphics indicate Intel 4400 and work well, sound works, but I am now having trouble with my wifi and bluetooth adapter. I use a AzureWave Broadcom BCM94352 HMB 802.11/ac/867Mbps WLAN + BT4.0 Half Mini PCI-E and had previously patched it and they were working fine on 10.10.1 with the master fork EFI.zip. Any idea of what could have caused them to stop working with the new EFI kexts and DSDT/SSDTs?

Frank

**Comment by gvkt** [comment]
I don't think it has anything to do with the DSDT/SSDT patches since there has only been a very small localized change in one SSDT. It is most likely a kext conflict. You did keep a back up of the previous installation right?

I don't know if this card worked OOB or there was anything special done for it. My Realtek wifi card was not affected by this change.

Just walk through the installation again and make sure you have not left out anything. Sometimes it is missing a step or unintended step.

Things to check, you removed the earlier Touchpad kext from /S/L/E before installing the latest one. You removed the SSDT-1.aml from the previous when you copied the SSDT-5.aml. You removed the kexts you installed via kext utility from the EFI->Clover->kexts->Others folder as well as any unused ones like atheros wifi kext, etc. Try to get back to the same set of kexts as earlier in the Clover kexts directory in the EFI partition as well as in /S/L/E.

The minimum required to fix the graphics problem is the SSDT-5.aml replacing SSDT-1.aml along with the config.plist changes. It should work fine for non graphic intensive use even without the FakePCIID kexts. So one way to do this is to start from the earlier Clover EFI configuration, make sure the card is working and just change the SSDT and the config.plist and nothing else from the new EFI.zip. If that works you can do the kext changes one step at a time and checking after rebooting for each change to see when the card stops working. That will isolate the kext that may be causing the problem.

Also check the Clover boot log to see if anything looks odd.

There is not much more help I can provide I am afraid.

**Comment by flp4** [comment]
That took care of my problem. Thanks!

**Comment by gvkt** [comment]
Great, glad it worked out. Can you post what caused the problem  just in case others face similar issue and stumble on to this thread? Thanks.

Let me know if you try the 10.10.2 update and have any issues.

**Comment by flp4** [comment]
I had no significant issues updating to 10.10.3 using the files on https://github.com/gvkt/Asus-Q501LA-Hackintosh. Just had to restore my audio kexts. Works good. I 
did see graphics artifacts on the menu bar until I refreshed my kexts with Kext Utility.

Frank

**Comment by gvkt** [comment]
There appears to be a way to not have to update AppleHDA kext after every update with a Clover patch. Have not had time to incorporate this. Will do so if and when I get time.
